docs/sources/dashboards/variables/_index.md

```diff
@@ -15,57 +15,78 @@ weight: 800
 
 # Variables
 
-{{< youtube id="mMUJ3iwIYwc" >}}
+A variable is a placeholder for a value.
+When you change the value, the element using the variable will change to reflect the new value.
 
-The following topics describe how to add and manage variables in your dashboards:
+Variables are displayed as drop-down lists at the top of the dashboard.
```

Comment:
Not all of them, there is the textbox variable also that is just an input in which you can add arbitrary values